README.md

```diff
@@ -27,6 +27,45 @@ python reparameterization.py --model_arch yolov7 --training_ckpt runs/train/yolo
 python save_state_dict.py --weights weights/yolov7_last.pt --save_path weights/yolov7_last_state.pt --cfg cfg/deploy/yolov7.yaml
 ```
 
+## Testing/Evaluating (with pycocotools)
+`test_coco.py` is a Python script for evaluating models with COCO ground truth annotations. Please follow the instructions below to use it effectively.
+
+### Prerequisites
+
+Before using `test.py`, ensure the following prerequisites are met:
+
+1. **`fdet-api`**: Download [fdet-api](https://github.com/yhsmiley/fdet-api/blob/master/PythonAPI/pycocotools/cocoeval.py).
+
+1. **COCO JSON File**: Provide a COCO JSON file containing ground truth annotations in the same folder as your image dataset. JSON image file names should match image file names.
+
+1. **Weight Reparameterization**: Note that you have to reparameterize the weights before testing (refer to [Reparameterization](#reparameterization-to-convert-weights-for-use-in-inference-branch).
+
+1. **COCO Annotation Format**: Annotations (e.g., `cat_id`, `ann_id`, `image_id`, etc.) must adhere to standard COCO format (start from 1).
+
+1. **`iscrowd` Annotation**: Ensure `iscrowd` exists in the ground truth annotations.
```

Comment:
The latest commit in `fdet-api` repo fixes this issue

Reply:
Yay nice! Ok will remove